From 03ecbd1229ba128aa759aedade468b28692c7334 Mon Sep 17 00:00:00 2001
From: Jan Wyszynski <jan.wyszynski@joyent.com>
Date: Tue, 15 Aug 2017 21:07:00 +0000
Subject: [PATCH] joyent/node-manta#318: node-manta nodejs version support
 Reviewed by: Angela Fong <angela.fong@joyent.com> Reviewed by: Dave Pacheco
 <dap@joyent.com>

---
 README.md | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/README.md b/README.md
index e581700..934f6c6 100644
--- a/README.md
+++ b/README.md
@@ -168,3 +168,14 @@ Here is how to cut a release:
 
    This will run a couple checks (clean working copy, versions in package.json
    and CHANGES.md match), then will git tag and npm publish.
+
+# Supported Node.js Versions
+
+Currently, node-manta officially supports functionality covered by all but the MPU tests 
+that live in the test directory at the project root.
+
+* v0.10.48
+* v0.12.18
+* v4.8.4
+* v6.11.2
+* v8.3.0
-- 
2.21.0

